URBNSwiftyConvenience/Classes/URBNBorders.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  URBNBorders.swift
+//  Pods
+//
+//  Created by Lloyd Sykes on 6/30/17.
+//
+//
+
+import Foundation
+
+extension UIView {
+    public var urbn_leftBorder: URBNBorder? {
```

Comment:
good point, property names on extensions needs to be unique _(no property overloading like there is with function overloading)_